src/hooks/protections/script_event_handler.cpp

```diff
@@ -459,8 +459,10 @@ namespace big
 		if (g.debug.logs.script_event.logs
 		    && (!g.debug.logs.script_event.filter_player || g.debug.logs.script_event.player_id == player->m_player_id)) [[unlikely]]
 		{
+			uint64_t rockstar_id = player->get_net_data() == nullptr ? 0 : player->get_net_data()->m_gamer_handle.m_rockstar_id;
```

Comment:
Can you move this to inside the player.cpp class? That way, it will enforce OOP and not expect the caller to handle this, as the caller doesn't care about the net data (well they do just not directly) they only want their Rockstar ID if we have it.